docs/actions.md

```diff
@@ -216,10 +216,17 @@ ok: invoked /whisk.system/samples/greeting with id 5975c24de0114ef2b5c24de0118ef
 A blocking invocation request will _wait_ for the activation result to be available. The wait period
 is the lesser of 60 seconds or the action's configured
 [time limit](reference.md#per-action-timeout-ms-default-60s).
-The result of the activation is returned if it is available within the wait period.
+
+The result of the activation is returned if it is available within the blocking wait period (this wait period is by default 60 seconds as well).
 Otherwise, the activation continues processing in the system and an activation ID is returned
 so that one may check for the result later, as with non-blocking requests
 (see [here](#watching-action-output) for tips on monitoring activations).
+If execution of the action exceeds the action's configured time limit within the blocking wait period
```

Comment:
If this is strictly intended to explain the activation's response status, then I suggest dropping the sentence on line 224 because what you write next is true for _all_ activations that exceed their configured duration.

Reply:
Unless the configured time limit is longer than 60 seconds and the blocking wait time is the default. In this case it returns 202 as you said previously? Curious...

Reply:
There are two things at play:
1. the HTTP response to the client: if the action has not finished in 60 seconds, the status code is 202 and an activation id is returned
2. in all cases (unless the system fails to generate one), the activation record for an action is created and recorded, and if the action timed out, the response status will be application error (for run) or developer error (for init).

Is that clearer?